CHT/Temperature.C

```diff
@@ -42,18 +67,21 @@ void preciceAdapter::CHT::Temperature::read(double * buffer)
 {
     int bufferIndex = 0;
 
+
     // For every boundary patch of the interface
     for (uint j = 0; j < patchIDs_.size(); j++)
     {
         int patchID = patchIDs_.at(j);
 
+
```

Comment:
There are a few additional empty lines in your changes. Does this come from clang-format?

Reply:
Usually not, I will remove them in the update.